gc/base/standard/ConcurrentGC.cpp

```diff
@@ -3382,7 +3405,7 @@ MM_ConcurrentGC::heapReconfigured(MM_EnvironmentBase *env, HeapReconfigReason re
 	 */
 
 	if ((lowAddress != NULL) && (highAddress != NULL)) {
-		Assert_MM_true(_rebuildInitWorkForAdd && (reason == HEAP_RECONFIG_EXPAND));
```

Comment:
This condition is no longer valid - `_rebuildInitWorkForAdd` may be rest by tuneToHeap.